Add a link to the quick start guide (#4)
tyntec/botbuilder-adapter-tyntec-whatsapp: README.md

});
```

> The full quick start guide can be found in the [tyntec Docs Center](https://www.tyntec.com/docs/whatsapp-business-api-integration-microsoft-bot-framework).

Reviewer: ```suggestion
You can read a quick start guide in the [tyntec Docs Center](https://www.tyntec.com/docs/whatsapp-business-api-integration-microsoft-bot-framework).
```

Reviewer: I would suggest using the active voice here :)

Author: Like this: https://github.com/tyntec/botbuilder-adapter-tyntec-whatsapp/commit/4684ecea41632387405ccbce231be91141c5d4bf ?
